packages/animations/lib/src/shared_axis_transition.dart

```diff
@@ -48,11 +61,11 @@ import 'utils/curves.dart';
 ///         ),
 ///       );
 ///     },
-///     '/a': (BuildContext context) {
+///     '/a' : (BuildContext context) {
 ///       return Container(
 ///         color: Colors.blue,
 ///         child: Center(
-///           child: RaisedButton(
+///           child: MaterialButton(
```

Comment:
I believe MaterialButton is to be avoided and RaisedButton, FlatButton, or OutlineButton should be used.